scripts/build/build/targets.py

```diff
@@ -206,7 +206,20 @@ def BuildEfr32Target():
     target.AppendModifier('rpc', enable_rpcs=True)
     target.AppendModifier('with-ota-requestor', enable_ota_requestor=True)
     target.AppendModifier('sed', enable_sed=True)
-    target.AppendModifier('low-power', enable_low_power=True)
+    target.AppendModifier('low-power', enable_low_power=True).OnlyIfRe('-sed')
+    target.AppendModifier('chip_build_libshell', chip_build_libshell=True)
```

Comment:
Consider simplifying suffixes. At least having `chip_` prefix everywhere seems to stutter.

How about `with_shell` or `shell` and `no_logging`  and such?

Reply:
agreed, names were changed to no_logging and shell.